game_event/models.py

```diff
@@ -10,7 +10,6 @@
 
 
 class GameEvent(models.Model):
-    id = models.IntegerField(primary_key=True)
```

Comment:
I see you mentioned there where some conflicts of the id between Django and this model.
Can you please explain me: do you mean that there is an id field built-in already in the game_event model?
and we don't need to mentioned the id field explicit like before?